scripts/sign_verify_file_ssh.sh

```diff
@@ -102,19 +105,24 @@ if [ "$MODE" == "sign" ]; then
     convert_private_key "$PRIVATE_KEY" "$TEMP_KEY"
 
     echo "Signing the file..."
-    ssh-keygen -Y sign -f "$TEMP_KEY" -P "${KEY_PASSPHRASE:-}" -n file "$FILE_TO_SIGN"
-
-    [ ! -f "$SIG_FILE" ] && { echo "Error: Signing failed."; exit $FILE_PROBLEM; }
-    echo "Signature created: $SIG_FILE"
-
-    cat <<EOF
+    if [[ -n "${NAMESPACE}" ]]; then
+        ssh-keygen -Y sign -f "$TEMP_KEY" -P "${KEY_PASSPHRASE:-}" -n "${NAMESPACE}" "$FILE_TO_SIGN"
+        cat <<EOF
 
 For verification, your allowed signers file could contain:
-identity_1 $(cat "${TEMP_KEY}.pub")
+identity_1 namespaces="${NAMESPACE}",valid-before="LAST_VALID_DAY" $(cat "${TEMP_KEY}.pub")
 EOF
+    else
+       # refuse to sign without a namespace
+       echo "Error: missing namespace for signing"
+       exit $MISSING_NAMESPACE
```

Comment:
Not sure why this needs to be an error, why not use a default like `file`? This is potentially not enforced by the allowed signers file anyway (namespace is an optional restriction).

I understand that _we_ want to leverage namespace, but other users of the bot may not. Again, I can implement in the follow-up PR.